docs/consume-packages/managing-the-global-packages-and-cache-folders.md

```diff
@@ -0,0 +1,116 @@
+---
+title: How to manage the global packages, cache, temp folders in NuGet | Microsoft Docs
+author: kraigb
+ms.author: kraigb
+manager: ghogen
+ms.date: 03/19/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: How to manage the global package installation folder, the package cache, and the temp folders that exist on a computer, which are used when installing, restoring, and updating packages.
+keywords: NuGet global packages folder, NuGet package cache, package caching, package installation folder, NuGet caches, managing caches, local NuGet cache, global NuGet cache, NuGet locals command, clearing a cache
+ms.reviewer:
+- karann-msft
+- unniravindranathan
+---
+
+# Managing the global packages, cache, and temp folders
+
+Whenever you install, update, or restore a package, NuGet manages packages and package information in several folders outside of your project structure:
+
+| Name | Description and Location (per user)|
+| --- | --- |
+| global&#8209;packages | The *global-packages* folder is where NuGet installs any downloaded package. Each package is fully expanded into a subfolder that matches the package identifier and version number. Projects using the PackageReference format always use packages directly from this folder. When using the `packages.config`, packages are installed to the *global-packages* folder, then copied into the project's `packages` folder.<br/><ul><li>Windows: `%userprofile%\.nuget\packages`</li><li>Mac/Linux: `~/.nuget/packages`</li><li>Override using the NUGET_PACKAGES environment variable, or the `globalPackagesFolder` [configuration setting](../reference/nuget-config-file.md#config-section) (PackageReference only). The environment variable takes precedence over the configuration setting.</li></ul> |
```

Comment:
It will be interesting to also mention `RestorePackagesPath` which is also another MSBuild property way to override global packages folder.

Reply:
OK